weatherman.py

```diff
@@ -0,0 +1,47 @@
+from Report import Report
+from Reader import Reader
+from Parser import Parser
+import argparse
+import os
```

Comment:
PEP8 violation, import should be in alphabetical order and categorised into three groups

```
built-in imports

third party imports

local imports
```